pkg/model/components/cilium.go

```diff
@@ -139,6 +139,10 @@ func (b *CiliumOptionsBuilder) BuildOptions(o *kops.Cluster) error {
 		c.EnableL7Proxy = fi.PtrTo(true)
 	}
 
+	if c.EnableLocalRedirectPolicy == nil {
+		c.EnableLocalRedirectPolicy = fi.PtrTo(false)
+	}
+
```

Comment:
Please remove this change. It should not affect any of the output.

Reply:
@hakman I made a mistake I will create another PR and fix it

Reply:
no worries. looking forward to it.